src/frontend/src/lib/utils/onramper.utils.ts

```diff
@@ -59,3 +62,30 @@ const toQueryString = (params: Omit<BuildOnramperLinkParams, 'wallets'>) =>
  */
 export const buildOnramperLink = ({ wallets, ...params }: BuildOnramperLinkParams) =>
 	`${ONRAMPER_BASE_URL}?apiKey=${ONRAMPER_API_KEY}&${toQueryString(params)}&wallets=${walletsToParam(wallets)}`;
+
+/** Map a list of tokens to a list of Onramper wallets.
```

Comment:
What's a wallet in the context of onRamper? Wallet like a wallet?

Reply:
in the official Onramper documentation, they call it wallet, but it is essentially the wallet address, or in general the address to send the tokens to. It is constituted by the crypto ID and the wallet address

Reply:
Thanks. Worth mentionning in the comment.